Composer/packages/electron-server/src/electronWindow.ts

```diff
@@ -43,7 +42,7 @@ export default class ElectronWindow {
     if (isLinux() && !isDevelopment) {
       // workaround for broken .AppImage icons since electron-builder@21.0.1 removed .AppImage desktop integration
       // (https://github.com/electron-userland/electron-builder/releases/tag/v21.0.1)
-      browserWindowOptions.icon = join(getUnpackedAsarPath(), 'resources/composerIcon_1024x1024.png');
+      browserWindowOptions.icon = join(__dirname, '../resources/composerIcon_1024x1024.png');
```

Comment:
Just making sure: this path resolves correctly in the packaged app as well?

Reply:
yes, images are fine with asar